app/views/organizations/subjects/_subject.html.slim

```diff
@@ -0,0 +1,10 @@
+= turbo_frame_tag subject, class: "card"
+  .card-title = subject.name
+  - if allowed_to?(:update?, with: SubjectPolicy)
```

Comment:
Shouldn't it be
```suggestion
  - if allowed_to?(:update?, subject)
```
?

Reply:
I think nope, because this record not using in policy yet

Reply:
But it is more readable?